core/modules/layout/plugins/access/entity_bundle_layout_access.inc

```diff
@@ -70,10 +70,15 @@ class EntityBundleLayoutAccess extends LayoutAccess {
   function checkAccess() {
     if (!$this->settings['bundles']) {
       return FALSE;
-    };
+    }
 
     /* @var Entity $entity */
     $entity = $this->contexts[$this->entity_type]->data;
+    // Make sure we're testing an entity here. Views could override system
+    // paths and cause difficulties. See issue #5809.
```

Comment:
When viewing this via GitHub diff online, the hah-tagged issue number is linkified - not the same when someone might be viewing the inline comments with a text editor. May I suggest that we actually use the link to the issue here?:

```suggestion
    // paths and cause difficulties.
    // See https://github.com/backdrop/backdrop-issues/issues/5809
```